### ISSUE TITLE
Remove --verbose flag from zebrad

### ISSUE THREAD

**Issue by oxarbitrage** [ZcashFoundation/zebra#831]
By the merge of https://github.com/ZcashFoundation/zebra/pull/827 the verbose flag will no longer do anything however will still show up in help output.

Removing the flag is blocked by Abscissa changes(https://github.com/ZcashFoundation/zebra/pull/827#discussion_r465835843) however this ticket is to work on it in the future.

**Comment by teor2345** [comment]
Marking this as a bug, because the usage message does not match the behaviour of zebrad.

**Comment by Rqnsom** [comment]
Hi there, I think this might be a related issue:

Initially, I wanted to check the description of the `--verbose` option since one of our scripts in the Ziggurat project uses that option.

I had to find the zebrad's help output, but to find out what the `help` subcommand looks like, I just used an invalid subcommand:
```sh
➜  codecs git:(main) ✗ zebrad random-things
error: unrecognized command `random-things`

zebrad 1.0.0-rc.0
Zcash Foundation <zebra@zfnd.org>

FLAGS:
    -c, --config CONFIG       path to configuration file
    -h, --help                print help message
    -v, --verbose             be verbose
```

After seeing `    -h, --help                print help message`,  I used `--help` with no effect. Not only did I not get the help text, it even started the process.
```sh
➜  codecs git:(main) ✗ zebrad --help
2022-11-13T12:20:45.661166Z  INFO zebrad::components::tracing::component: started tracing component filter="info" TRACING_STATIC_MAX_LEVEL=LevelFilter::INFO LOG_STATIC_MAX_LEVEL=Info
2022-11-13T12:20:45.661203Z  INFO zebrad::application: Diagnostic Metadata:
version: 1.0.0-rc.0
Zcash network: Mainnet
state version: 25
branch: master
git commit: a4cb835
commit timestamp: 2022-10-12T01:17:55Z
target triple: aarch64-apple-darwin
build profile: release
[...]
```
Then I realized the command `zebrad help` would give me what I needed. And there I also saw that the `verbose` flag doesn't exist anymore.

It's an interesting issue with the obsolete help text, which provides wrong info.
Should I create a new issue for this, or?

**Comment by oxarbitrage** [comment]
Thanks for the report @Rqnsom , some of the behavior is probably related to https://github.com/ZcashFoundation/zebra/pull/4957 where we changed to start zebra by default. 

 > Should I create a new issue for this, or?

Yea, please create or let me know and i can just create it for you.

**Comment by Rqnsom** [comment]
Thanks @oxarbitrage,
I created a new issue [here](https://github.com/ZcashFoundation/zebra/issues/5624).
Please let me know if you need more info or anything else
